### DISCUSSION TITLE
Release v1.4.0 / 5.59.0

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2292] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This is a unsigned test build (that is the driver is signed but not the installers), it improves the SandMan UI a lot and adds various new UX oriented features, it also makes the Sandboxie Plus code base Qt6 compatible, that said, the build currently uses still Qt5, a switch to Qt6 is planned for later. With regard to Qt6's no longer present Windows 7 support, this won't be an issue as a custom build of Qt6's base components will be used which are Windows 7 compatible.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Added
- added integrated run from start menu [#1836](https://github.com/sandboxie-plus/Sandboxie/issues/1836)
- added start menu enumeration [#1570](https://github.com/sandboxie-plus/Sandboxie/issues/1570)
- added UI for breakout processes [#1904](https://github.com/sandboxie-plus/Sandboxie/issues/1904)
- added option to customize double-click action per sandbox
- added new miscellaneous tab in the advanced options tab allowing to configure specific processes and other advanced options
- added "SeparateUserFolders=y" and "SandboxieLogon=y" to the sandbox options dialog
- added icons to the section labels on the option pages

### Changed
- prepared for Qt 6.3.1
- restructured the general settings page
- restructured the sandbox options page, added a new tab and moved some of the advanced options there

### Fixed
- fixed menu bar issue in Plus UI [#2280](https://github.com/sandboxie-plus/Sandboxie/pull/2280) (thanks okrc)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.4.0'>Release v1.4.0 / 5.59.0</a>.</em>

**Comment by BilleBarrett** [comment]
I got this error and I am not sure what was checked and where it is, could you help me
"The program UseRuleSpecifity stated in the box DefraultBox will terminate in 5 minutes because the box was configured to use features available to project supporters. Then a request to be a supporter 
I have no problem with being a supporter but what did I check on a new install that caused this
Thanks
Bill e. Barrett

**Reply by APMichael** [reply]
I suppose you have enabled the following setting:

![sandboxie_plus](https://user-images.githubusercontent.com/33252157/193617840-6056f7e8-5f50-4f3b-9a69-10fc4e92673f.jpg)
